### DISCUSSION TITLE
Database connection error

### DISCUSSION THREAD

**Discussion by 44elovekk** [akiver/cs-demo-manager#738] [Q&A]
I installed everything correctly, even uninstalled and installed again, it still gives this error. And PostgreSQL is running.
![image](https://github.com/akiver/cs-demo-manager/assets/67231266/8faf1613-6c37-41c7-8290-f1c70823e089)

**Comment by akiver** [comment]
What's your OS language?

Could you please upload the log files (it's located in C:/Users/USERNAME/.csdm/logs)?
